content/attacks/posts/2016-08-02-Bitfinex.md

```diff
@@ -0,0 +1,37 @@
+---
+date: 2016-08-02
+target-entities: Bitfinex
+entity-types: Exchange
+attack-types: Custodian
```

Comment:
Attack type for the case is `Web Infrastructure Attack'

Reply:
Done